### ISSUE TITLE
Journal Manager role/permission is unclear 

### ISSUE THREAD

**Issue by pmangahis** [pkp/pkp-lib#5962]
Feedback from a hosting client new to OJS 3 found that having the Journal Manager role/permission displayed with no roles enabled and greyed out to be confusing.

![image](https://user-images.githubusercontent.com/50342731/83678397-8bf32f00-a592-11ea-8381-6d7eaea450af.png)
 
To a new user, this could suggest that the JM doesn't have permission within those areas of the workflow. As well, having the Journal Editor inherit this as a permission level of Journal Manager might suggest that they don't have permission either as it doesn't match what they have checked on their role. Having Journal Editor permissions checked and Journal Manager permission unchecked seemed counter-intuitive

It might make it more clear to have the boxes for the JM role as well.

**Comment by AhemNason** [comment]
If you "edit" the journal manager roll and just save it, the checks will fill in. I just did this on our 3.2.1-1 install.

**Comment by NateWr** [comment]
I agree this is confusing. Unfortunately, this is a case where the checkboxes really do mean something, even if it's not apparent at first. When assigning a participant to a submission, you can search by one of the roles for the stage. 

So when assigning a participant in the submission stage, you will only be able to assign a role that has a checkbox for the Submission stage. In the example above, that means that you can assign a Journal Editor but not a Production Editor or Journal Manager.

I'm not entirely sure how to address this. The whole stage assignment thing is not terribly intuitive, but upending that would be a lot of work. I suggest that we leave the functionality as it is and try to improve the clarity of the role settings. But I've not get any ideas for precisely how to do that without a significant rewrite of the roles grid.

I'm going to pull the Try Me label off of this for now until we can land on what we want to happen here.
